src/show/components/TextField/index.js

```diff
@@ -1,23 +1,83 @@
-import React from "react";
-
+import React, { useState } from 'react';
+import { ReactComponent as EyeIcon } from '../../assets/svg/eye-icon.svg';
 const TextField = ({
+  name,
+  type = 'text',
   label = "",
   placeholder = "",
-  required = false,
   value = "",
-  password = false,
-}) => (
-  <div>
-    <h6>{label}</h6>
-    <input
-      placeholder={placeholder}
-      type={password ? "password" : "text"}
-      password={true}
-      className="bg-gray-100 text-black w-64 px-2 rounded border-2 border-black"
-    //   value={value}
-    />
-    <h6 className="text-xs text-right">{required && "required"}</h6>
-  </div>
-);
+  onChange,
+  hasEye,
```

Comment:
give defaults values to all props. if possible install this library and start giving types as well at the end something like this 

TextField.propTypes = {
name :  PropTypes.string,
type : PropTypes.oneOf[{'text', 'password'}],
onChange : PropTypes.func
}

export default TextField;

Reply:
this should be done in every functional component . if we create a component or we create a page . Defaults and proptypes is a good practise